app/lib/package/models.dart

```diff
@@ -152,6 +152,10 @@ class Package extends db.ExpandoModel<String> {
   @db.StringProperty(indexed: false)
   String? automatedPublishingJson;
 
+  /// The JSON-serialized format of the [AutomatedPublishingLock].
+  @db.StringProperty(indexed: false)
+  String? automatedPublishingLockJson;
```

Comment:
Wouldn't it be better to merge these two properties into one property?

I'm thinking that `AutomatedPublishing` being a different type from the one we have in the API is probably a good idea too, that might help us not making mistakes in the future.